### ISSUE TITLE
Allow empty pdnsmgrd URL

### ISSUE THREAD

**Issue by stbuehler** [PowerDNS/pdnscontrol#104]
and disable the buttons that rely on it; for simple zone managing pdnsmgrd is not needed.

**Comment by zeha** [comment]
Agreed; for now the workaround is to use the same URL for both (and then stuff fails).
